docs/new-components.md

```diff
@@ -0,0 +1,210 @@
+we require that you **build** your component outside of the opentelemetry-collector-contrib
+repository. This is also the fastest way to start using your component and to publish it for others to
+consume if you want to. 
+
+A component is a Go module (library) built using the
+`go.opentelemetry.io/collector` set of libraries. These libraries contain examples (e.g. see the
+example on the [`go.opentelemetry.io/collector/exporter` module
+documentation](https://pkg.go.dev/go.opentelemetry.io/collector/exporter)). The official
+ documentation also has [a section on how to build various kinds of
+components](https://opentelemetry.io/docs/collector/building/). You can also use existing
+implementations on this repository as a reference. The key criteria to implement a component is to:
+
+* Implement the
+  [component.Component](https://pkg.go.dev/go.opentelemetry.io/collector/component#Component)
+  interface
+* Provide a configuration structure which defines the configuration of the component
+* Provide the implementation which performs the component operation
+
+To **use** your component you can [use the OpenTelemetry Collector
+Builder](https://opentelemetry.io/docs/collector/custom-collector/). Even if you don't publish your
+component, you may [specify a local folder using the `replaces` option of the
+builder](https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder#configuration)
+to include it in your build.
+
+If you want to **publish** your component for other people to try it out, you can do so for free
+using Github or other git forges. To do so, you need to [publish your component as a Go
+module](https://go.dev/doc/modules/publishing). You can publish multiple components from a single
+repository by including the path to the component in the tag: for example, the [`filelog`
+receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver#file-log-receiver)
+v0.139.0 version is [available as a Go
+module](https://pkg.go.dev/github.com/open-telemetry/opentelemetry-collector-contrib/receiver/filelogreceiver@v0.139.0)
+because we pushed the
+[`receiver/filelogreceiver/v0.139.0`](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver)
+git tag to this repository. We recommend you make a release at least every time there is a breaking
+change on any of the Go modules you depend on.
+
+Finally, to **distribute** your component and make sure others can easily discover it, [add it to
+the OpenTelemetry registry](https://opentelemetry.io/ecosystem/registry/adding/).
+
+If you think your component fits the requisites to be on the opentelemetry-collector-contrib
+repository, you may choose to donate it following the steps on the next section. Your activity,
+community and popularity within your own repository will help you make the case for the component to
```

Comment:
I'm not following the wording of "community and popularity", should there be a comma after "community" to make it more clear?